wordpress-mobile/WordPress-Android
# Missing indicators in pages list

This PR fixes a missing functionality where there were not indicators for `Homepage` and `Posts Page` on the pages list. I'm adding these indicators into the field that previously only contained the date the page was created.

To test:
- Go to the pages list on a WP.com or Jetpack site
- Notice that there are Homepage/Posts Page indicators visible
- Try to set Homepage/Posts Page
- The indicators are updated

To test 2:
- The indicators are not visible on a self-hosted site without a Jetpack

![homepage_in_page_list](https://user-images.githubusercontent.com/1079756/83636889-0037cd80-a5a7-11ea-8bb6-1505ec12ac80.gif)


PR submission checklist:
